packages/logs/src/boot/startLogs.spec.ts

```diff
@@ -170,6 +170,31 @@ describe('logs', () => {
       expect(server.requests.length).toEqual(0)
       expect(sendSpy).toHaveBeenCalled()
     })
+
+    it('should send console logs if ff "forward-logs" enabled', () => {
+      const errorLogger = new Logger(noop)
+      const logErrorSpy = spyOn(errorLogger, 'log')
+
+      updateExperimentalFeatures(['forward-logs'])
+      startLogs({ errorLogger })
+
+      /* eslint-disable no-console */
+      console.log('foo', 'bar')
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏not sure if it is from there but it seems that a couple of `'foo', 'bar'` are now displayed on the unit test logs, it could be nice to get rid of them

```
XXX:INFO [launcher]: Starting browser ChromeHeadless
XXX:INFO [Chrome Headless YYY]: Connected on socket ZZZ with id 0
LOG LOG: 'foo', 'bar'
LOG LOG: 'foo', 'bar'
XXX:INFO [jasmine-seed-reporter]: Chrome Headless YYY: Randomized with seed 1

TOTAL: 1155 SUCCESS
```

Reply:
Yep, sure. Done